html/modules/custom/hr_paragraphs/migrate/README.md

```diff
@@ -92,6 +92,14 @@ managers of a cluster are added as member of the operation.
 drush hr_paragraphs:import-members --verbose
 ```
 
+### Users
+
+All existing users on the old D7 site need to be migrated, since we use the same uid on both sites.
+
+```bash
+hr_paragraphs:import-users --skip-existing --account-active|--account-blocked --ids=1,2,3 --group-ids=7,8,9
```

Comment:
I don't seem to be using the command correctly. Does the file need to be in a specific location, as opposed to specifying it as an argument?

```sh
# no IDs specified
$ fin drush hr_paragraphs:import-users --skip-existing --account-active ~/Downloads/membership.tsv

  Too many arguments, expected arguments "command".

# try specifying the first ID found in TSV file
$ fin drush hr_paragraphs:import-users --skip-existing --account-active --ids=96 ~/Downloads/membership.tsv

  Too many arguments, expected arguments "command".

```

Reply:
File needs to be in /var/www/tsv or set it in config, `hr_paragraphs.settings.csv_source_dir`